service/src/main/java/io/aklivity/zillabase/service/internal/util/ZillabaseAuthUtil.java

```diff
@@ -24,15 +24,15 @@
 import jakarta.json.JsonObject;
 import jakarta.json.JsonReader;
 
-public class ZillabaseSsoUtil
+public class ZillabaseAuthUtil
```

Comment:
Let's rename this as `ZillabaseAuthHelper` since util classes are generally only static methods.